wdn/templates_5.3/js-src/banner.babel.js

```diff
@@ -0,0 +1,45 @@
+// WDN Notice Banner Display
+function displayWDNNoticeBanner() {
+    var bannerEnabled = true;
+    if (bannerEnabled) {
+        var messageContent = false;
+        var messageSessionKey = 'wndNoticeMessage';
+        var sessionContent = sessionStorage.getItem(messageSessionKey);
+
+        // clear saved message every five minutes
+        setInterval(function(){ sessionStorage.removeItem(messageSessionKey); }, 300000);
```

Comment:
why don't we bump this up to like 15 (?) minutes

Reply:
i bumped it to 15 and changed how to calcuates the minutes to milliseconds for easier readability